decentralised-identity.md

```diff
@@ -0,0 +1,106 @@
+- Privacy: SSI is designed to keep your information private. You're not part of a massive database that could be hacked. Instead, your data is secure, often using advanced technologies like blockchain.
+- No Middlemen: Traditional systems often involve middlemen, like banks verifying your identity. SSI cuts out the middle, allowing direct interactions between you and whoever needs to confirm your identity.
+- Flexibility: Your digital identity should work across different platforms. SSI aims to be flexible and usable across various services without needing to create a new identity for each one.
+
+In essence, SSI puts you in the driver's seat of your digital identity, emphasizing control, privacy, and simplicity.
+
+### Blockchain Technology: 
+
+Decentralized identity often uses blockchain tech. Blockchain is like a secure digital ledger, making sure no one messes with your identity info. It's resistant to tampering, so your data stays safe. Plus, it's transparent, meaning you can easily see what's happening. With blockchain, you're in charge of your digital identity and transactions.
+
+Here's how it plays a key role:
+
+1. Secure Digital Ledger: Blockchain acts like a super-secure, digital ledger or record-keeping system. Information about your identity is stored in blocks, and these blocks are linked together in a chain. Once a piece of data is in a block, it's tough to change without altering all the subsequent blocks, making it tamper-resistant.
+2. Data Integrity: The tamper-resistant nature of blockchain ensures the integrity of your identity data. Once your information is recorded, it's practically impossible for someone to sneak in and make unauthorized changes without everyone noticing. This helps in maintaining the accuracy and reliability of the information.
+3. Transparency: Blockchain is transparent by design. Every participant in the network has access to the same information. This transparency ensures that there's a shared view of the data, reducing the chances of disputes or discrepancies.
+4. Decentralization: Traditional systems often rely on a central authority to manage identity data. In blockchain-based decentralized identity, there's no single point of control. Instead, the network is distributed across many computers (nodes), making it more resilient and less susceptible to hacking or unauthorized access.
+5. User Control: Blockchain empowers individuals by giving them control over their private keys. These cryptographic keys allow you to access and manage your identity on the blockchain. As the owner of your keys, you have the ultimate say in who gets to see your data and when.
+6. Enhanced Security: The use of cryptographic techniques in blockchain ensures a high level of security. Information is encrypted and tied to your digital identity, making it difficult for unauthorized parties to gain access.
+
```

Comment:
Very good point, yes I see how it can come across as 'hey this is the ideal solution' . Going to spend today rewriting it to make it less bias

Reply:
I've removed a lot of the identity repetition and used DID instead of Decentralised Identity. I've added a paragraph to counteract the bias with these statements. It lists the drawbacks.